src/seedu/addressbook/data/person/Email.java

```diff
@@ -5,29 +5,27 @@
 /**
  * Represents a Person's email in the address book.
  * Guarantees: immutable; is valid as declared in {@link #isValidEmail(String)}
+ * Inherits methods from Contact class
```

Comment:
Glad you made the effort to update class headers =).
Though this comment line seems to not be required.
As it can be read in the javadocs itself.